desktop/apps/tag/client.js

```diff
@@ -1,23 +1,37 @@
+import qs from 'querystring'
+import { data as sd } from 'sharify'
+import { assign } from 'lodash'
+
 import React from 'react'
 import ReactDOM from 'react-dom'
-import * as Relay from 'react-relay'
-import { data as sd } from 'sharify'
+import { Contents } from '@artsy/reaction-force/dist/Components/Tag'
+import { ContextProvider } from '@artsy/reaction-force/dist/Components/Artsy'
+
+import Tag from '../../models/tag.coffee'
+import CurrentUser from '../../models/current_user.coffee'
+
+// Update URL with current filters/mode/sort, for ease of sharing.
+const onStateChange = ({ filters, sort }) => {
+  const params = assign({ ...filters, sort }, {})
```

Comment:
You really only need `{ ...filters, sort }` here. I’m normally all for following existing patterns in code-bases, but in this case I’d say that if there are other/many places in Force where unneeded merging is done we should just remove that. Trying to read it honestly takes slightly more time because I assume it’s merging for a good reason and not immediately seeing the reason.

Reply:
I'll go in and switch these today.

Reply:
Thanks 🙏🏽